leaderless/node/app/main.py

```diff
@@ -16,6 +19,13 @@ class Transfer(BaseModel):
   to_id: str
   amount: int = Field(ge=0)
 
+class PrepareMessage(BaseModel):
+  propose_id: int
+
+class AcceptMessage(BaseModel):
+  propose_id: int
+  val: str
```

Comment:
These messages have to contain the paxos run ID / key of the entry in distributed log that corresponds to the paxos run.

Reply:
https://docs.google.com/document/d/1eEtBF9y2iqq1oMrBGdxKaPajgFim1L0GX0yzB409Al0/edit#